mozilla-mobile/android-components/components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -36,6 +36,19 @@
       </ul>
     ]]></string>
 
+    <!-- The text shown inside the advanced button used to expand the advanced options. It's only shown when a website has an invalid SSL certificate. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_advanced">Advanced…</string>
+    <!-- The advanced certificate information shown when a website sends has an invalid SSL certificate. The %1$s will be replaced by the website URL. It's only shown when a website has an invalid SSL certificate. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_techInfo"><![CDATA[
+        <label>Someone could be trying to impersonate the site and you should not continue.</label>
+        <br><br>
+        <label>Websites prove their identity via certificates. Firefox does not trust <b>%1$s</b> because its certificate issuer is unknown, the certificate is self-signed, or the server is not sending the correct intermediate certificates.</label>
+    ]]></string>
+    <!-- The text shown inside the advanced options button used to go back. It's only shown if the user has expanded the advanced options. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_back">Go Back (Recommended)</string>
+    <!-- The text shown inside the advanced options button used to bypass the invalid SSL certificate. It's only shown if the user has expanded the advanced options. -->
+    <string name="mozac_browser_errorpages_security_bad_cert_accept_temporary">Accept the Risk and Continue</string>0
```

Comment:
Is the 0 at the end expected?

Reply:
@keianhzo Could you please take a look here and at Flod's comments above, and help update? thanks!

Reply:
Thanks for that. Fixed, waiting for review: https://github.com/mozilla-mobile/android-components/pull/5624